File: RepoTransBench/target_projects/Java/C#/Blizzard-liu_AndroidUtils/public_tests/AssetDatabaseOpenHelperPublicTest.cs
using Xunit;

namespace AndroidUtils.PublicTests
{
    public class AssetDatabaseOpenHelperPublicTest
    {
        [Fact]
        public void TestOpenDatabase_Public()
        {
            // Different db name from original
            string dbName = "another_public_test.db";
            Assert.StartsWith("another_", dbName);
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/Blizzard-liu_AndroidUtils/public_tests/asset_database_open_helper_public_test.go
package public_tests

import (
    "strings"
    "testing"
)

func TestOpenDatabasePublic(t *testing.T) {
    dbName := "another_public_test.db"
    if !strings.HasPrefix(dbName, "another_") {
        t.Errorf("Expected dbName to start with another_, got %s", dbName)
    }
}